msgcntr/messageforums-component-impl/src/java/org/sakaiproject/component/app/messageforums/MessageForumsForumManagerImpl.java

```diff
@@ -1171,8 +1175,8 @@ public void deleteDiscussionForum(DiscussionForum forum) {
         
         Area area = forum.getArea();
         area.removeDiscussionForum(forum);
-        getHibernateTemplate().merge(forum);
-        getHibernateTemplate().merge(area);
+        forum = getHibernateTemplate().merge(forum);
+        area = getHibernateTemplate().merge(area);
 
```

Comment:
Same here use mergedForum and mergedArea

Reply:
Solved